cogs/dank_memes.py

```diff
@@ -142,18 +143,29 @@ class DankMemes(BaseCog):
             Dank Image
 
         """
+        if ctx.invoked_subcommand is None:
+            images = sorted(os.listdir(MEDIA_PATH))
+            if len(images) == 0:
+                await ctx.send('I don\'t have any images to shitpost with')
```

Comment:
I'd suggest switching this to double quotes to not have to escape the apostrophe

Reply:
```suggestion
                await ctx.send('I don\'t have any images to shitpost with')
```

Can do, This was how @michaelhillebrand had it and I didn't touch that line

Reply:
Gotcha, seems like a good time to switch it in my opinion, since we're already editing this section

Reply:
Done